stages/03_process_data.py

```diff
@@ -0,0 +1,48 @@
+# Imports
+# -------
+import re
+import pathlib
+import pandas as pd
+
+extensions = ['ism']
+extensions_re = re.compile(r'\.(' + '|'.join(re.escape(ext) for ext in extensions) + r')$')
+files = filter( lambda item: item.is_file(), pathlib.Path('download').rglob('*'))
+
+brick_dir = pathlib.Path('brick')
+brick_dir.mkdir(exist_ok=True)
+
+for file in files:
+
+    out_basename = re.sub(extensions_re, '.parquet', file.name )
+    out_file = brick_dir / file.relative_to('download').with_name( out_basename )
+
+    if file.match('*.ism'):
+
+        try:
+            protein_dir = str(file).split('/')[1]
+
+            names = {
```

Comment:
Also, for column names, it will make it easier for downstream tools if they do not contain spaces (less quoting).

Reply:
Resolved in dcd069665db995cda7971d6ffd8047396f46b380